src/core/tritonserver.h

```diff
@@ -667,6 +667,16 @@ TRITONSERVER_InferenceRequestOutputShape(
     TRITONSERVER_InferenceRequest* inference_request, const char* name,
     const int64_t** shape, uint64_t* dim_count);
 
+/// Get the names of all output tensors.
```

Comment:
"... in the result" to be more accurate. The function is actually returning the name of the requested outputs